release-operator/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
```

Comment:
@hannobraun shall I align the formatting style and version pinning with the `Cargo.toml` in the root?

Reply:
As you prefer. I like to do it my way, but that is a weak preference. No reason to impose my rules.